jeanmarieq.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html lang="pt-br">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Rock em espanhol</title>
+</head>
+<body>
+    <header>
+        <h1>Bandas de Rock em espanhol Clasicos</h1>
+        <img src="imagens/silueta.jpg" width="600" alt="silueta.">
+        <p><strong>Quem não gosta de boa música? Acho que todos nós temos um gênero musical que amamos, <em>certo?</em></strong></p>
+        <p>Em espanhol existe uma grande variedade de produções de rock and roll, para todos os gostos. </br>
+            Abaixo uma amostra de algumas bandas do rock espanhol que você deve conhecer</p>
+    </header>
+         <div>
```

Comment:
Utilizar divisor da página para contemplar o header independente

```suggestion
         <section>
```